### ISSUE TITLE
Reverse the order? 

### ISSUE THREAD

**Issue by levovix0** [treeform/fidget#160]
In Figma, "on top - on up" order looks normal, but in the code it is strange. For example, you have to iterate seq in reverse order.

**Comment by bung87** [comment]
yeah, the hacknews example also put the header rendering at bottom of code.
